statistics-production/api-data-standards.qmd

```diff
@@ -0,0 +1,191 @@
+
+The standards can be used to create individual filter columns or combined filters (i.e. breakdown_topic / breakdown_topic).
+
+Areas for which harmonised standards are currently available are:
+
+- [establishment / school / provider characteristics](../statistics-production/ud.html#establishment-characteristics)
+- [ethnicity](../statistics-production/ud.html#ethnicity)
+- [sex and gender](../statistics-production/ud.html#sex-and-gender)
+- [special educational needs](../statistics-production/ud.html#special-educational-needs)
+
+Areas which are currently under development are:
+
+- attainment metrics
+- disadvantaged status
+- free school meal status
+
+We encourage contributions to and feedback on all of the above and any other filter topic.
+
+### Examples of common non-standard filter col_names
+
+::: {.table-responsive}
+
+| Non-standard                | Potential standard equivalents     |
+|-----------------------------|------------------------------------|
+| ethnicity                   | ethnicity_major, ethnicity_minor   |
+| characteristic_sex          | sex                                |
+| school_type                 | establishment_type, establishment_type_group or education_phase |
+| pupil_sen_status            | sen_status                         |
+| characteristic_primary_need | sen_primary_need                   |
+| characteristic_topic        | breakdown_topic, breakdown_topic_establishment |
+| characteristic              | breakdown, breakdown_establishment |
+
+: Example non-standard col_names and their potential equivalents in the standardised framework.
+
+:::
+
+## Standardised indicator col_names
+
+Indicators should be named in line with the [indicator naming conventions set out in the open data standards](../statistics-production/ud.html#indicator-names).
+
+### Examples of common non-standard indicator col_names
+
+::: {.table-responsive}
+
+| Non-standard                           | Potential standard equivalents           |
+|----------------------------------------|------------------------------------------|
+| number_of_pupils                       | pupil_count                              |
+| NumberOfLearners, NumLearners          | pupil_count, learner_count               |
+| total_male, total_female               | pupil_count (plus sex filter)            |
+| pt_SEN_support                         | pupil_percent (plus SEN status filter)   |
+| num_provider, num_providers            | establishment_count                      |
+| no_schools, num_schools, total_schools | establishment_count                      |
+| num_inst, total_institutions, number_institutions, inst_count | establishment_count |
+
+: Example non-standard indicator col_names and their potential equivalents in the standardised framework.
+
+:::
+
+## Character limits for col_names and filter items
```

Comment:
What would you think to bumping this higher up? My thinking is purely as it's so short it's easy to scan and see there's stuff below rather than lost at the bottom